autosklearn/evaluation/__init__.py

```diff
@@ -86,10 +99,10 @@ def fit_predict_try_except_decorator(
 
 
 def get_cost_of_crash(
-    metric: Union[Scorer, List[Scorer], Tuple[Scorer]]
+    metric: Union[Scorer | Sequence[Scorer]],
```

Comment:
Like wise here, `Union[X | Y] == X | Y`, the `|` essentially is just the infix operator for `Union` in the same way you have `+` instead of `add(x, y)`.

i.e. `metric: Scorer | Sequence[Scorer]`

Reply:
Thanks for catching.